feat: cockerilltim changes to best practices (#6)
DesignSafe-CI/DS-User-Guide: user-guide/docs/curating/guides.md

### [Simulation](#simulation) { #simulation }

> We recommend reading the Best Practices Guide for [Simulation Data](../../bestpractices/working#simulationdata) as a prerequesite to initiating your project.

Author: Consider wording that is less verbose. User might understand faster. Also, the link was not working, I found this one to work.

```suggestion
Read the [Simulation Data Best Practices Guide](../../curating#bestpractices-simulationdata) prior to initiating your project.
```

Reviewer: excellent

#### [1. Add a Project](#simulation-step1) { #simulation-step1 }

> You can start a project at the very beginning of its lifespan, upload and curate data incrementally, then publish sets of data at your convenience.

Author: Test.
